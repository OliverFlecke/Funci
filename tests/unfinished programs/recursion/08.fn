main :: [Int] 
  = let ones :: [Int] = 1 : ones;
     in (head ones) : ((head (tail ones) : []);
