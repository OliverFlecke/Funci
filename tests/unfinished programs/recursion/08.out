Listy (Cons (Number (I 1) (Unit [])) (Cons (Number (I 1) (Unit [])) Empty))
